README.md

```diff
@@ -5,11 +5,6 @@
 Provides a public API for retrieving most mentioned people and connected people for the 'Six degrees' application.
 Connects to Neo4j to get the needed data.
 
-## Build & deployment
-
-* Build in DockerHub: https://hub.docker.com/r/coco/public-six-degrees/builds/
```

Comment:
Can you delete the DH job please?

Reply:
Can we leave it to the team to delete the dockerhub jobs?